docs/reference/cli/index.md

```diff
@@ -428,10 +428,10 @@ data-storage-mode: "archive"
 
 The strategy for handling historical chain data. Valid options are:
 
-- `minimal` - Stores the minimal required data to follow the chain and run validators. Finalized states and historic blocks are pruned.
-- `prune` - Stores all blocks, but finalized states are pruned.
```

Comment:
We should not remove this reference to prune mode. Teku still supports it. Minimal mode is the default, but users can choose to use Prune or Archive.